debug/terrain-debug.html

```diff
@@ -351,6 +351,11 @@
     }
 };
 
+map.on('idle', function() {
+    var elevation = map.queryTerrainElevationAt(map.getCenter());
```

Comment:
Carrying the original example from the issue in https://github.com/mapbox/mapbox-gl-js/pull/10597 that also makes use of this API.